src/ClimaAtmos.jl

```diff
@@ -75,6 +75,7 @@ include(
     ),
 )
 include(joinpath("prognostic_equations", "hyperdiffusion.jl"))
+include(joinpath("prognostic_equations", "closures.jl"))
```

Comment:
Can we give it a more specific name? This is about SGS closures, but right now just for mixing lengths, right? So why not `mixing_length_closure` or something like this?

Reply:
Yes, this is about SGS closures without EDMF. Right now it only has mixing length, but I was thinking about moving the buoyancy gradient, prandtl number, dycore vertical diffusivity, and maybe cloud fraction into this file as well. So basically it will contain all the closures that does not require EDMF variables. (EDMF closures, such as mixing length or non-hydrostatic pressure drag, are in a separate file `edmfx_closures.jl`.) Any suggestion on names?

Reply:
How about `gm_sgs_closures`? It's not perfectly specific, but maybe a little bit clearer?